MISSION.md

```diff
@@ -0,0 +1,98 @@
+Other unethical or unprofessional conduct.
+Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct. By adopting this Code of Conduct, project maintainers commit themselves to fairly and consistently applying these principles to every aspect of managing this project. Project maintainers who do not follow or enforce the Code of Conduct may be permanently removed from the project team.
+This code of conduct applies both within project spaces and in public spaces when an individual is representing the project or its community.”
+Instances of abusive, harassing, or otherwise unacceptable behavior in Open Cluster Management may be reported by contacting [NEED POINT OF CONTACT].” [[Reference](https://github.com/cncf/foundation/blob/master/code-of-conduct.md)]
+
+## Getting Involved
+
+Anyone who is interested in getting involved is welcome to contribute in a number of ways:
+
+Join the recurring meeting forums (see below) to provide input as a stakeholder and help validate proposed use cases.
+Suggest enhancements via github.com/open-cluster-management/enhancements for consideration to the community.
+Contribute to development via Pull Request for new enhancements or defect fixes.
+
+Suggested API and implementations will be accepted in accordance with the broad use cases outlined above. Our goal is to reserve the Kubernetes API Group open-cluster-management.io for well-reviewed and widely supported features.
+
+## Community Meeting Forum
+
+To ensure opportunities for broad user contributions, a public forum will be hosted to demonstrate new capabilities, solicit feedback and offer a forum for real time Q&A.
+Meeting recordings will be posted to a YouTube channel for offline viewing.
+
+- Public Agenda (Link TBD)
+- First Monday of the month
+  - 5-6 PM US Pacific / 8-9 PM US Eastern / 1-2 AM GMT / 8-9 AM (Tuesday) China
+- Third Monday of the month
+  - 8-9 AM US Pacific / 11 AM - 12 PM US Eastern / 4-5 PM GMT / 8-9 PM (Monday) China
+
+## Community Architecture Forum
+
+Technical discussions will also be held in an open forum. The topics for the Community Architecture Forum will cover technical review of APIs and relevant implementation details.
+Meeting recordings will be posted to a YouTube channel for offline viewing.
+
+
+- Public Agenda (Link TBD)
+- Second Monday of the month
+  - 5-6 PM US Pacific / 8-9 PM US Eastern / 1-2 AM GMT / 8-9 AM (Tuesday) China
+- Fourth Monday of the month
+  - 8-9 AM US Pacific / 11 AM -12 PM US Eastern / 4-5 PM GMT / 8-9 PM (Monday) China
+
+## Communication
+
+A public Google Group has been created to facilitate offline discussion outside of Github issues or Pull Requests.
+
+https://groups.google.com/forum/#!forum/open-cluster-management open-cluster-management@googlegroups.com
```

Comment:
Doesn't work?

Reply:
I will review and confirm that I have the permissions set correctly for the Group.